farsi_transcribe.py

```
@@ -0,0 +1,457 @@
+                'audio_duration': transcription['audio_duration'],
+                'processing_time': transcription['processing_time'],
+                'real_time_factor': transcription['rtf'],
+                'model': transcription['model'],
+                'chunks': transcription['chunks'],
+                'timestamp': time.strftime('%Y-%m-%d %H:%M:%S')
+            }, f, ensure_ascii=False, indent=2)
+        logger.info(f"Saved JSON: {json_path}")
+        
+    # Log summary
+    logger.info(f"\n{'='*50}")
+    logger.info(f"Transcription completed!")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)